_generated/AzureFunctionAppV1.versionmap.txt

```diff
@@ -1,2 +1,2 @@
-Default|1.236.0
-Node20_229_9|1.236.1
+Default|1.237.0
+Node20_229_9|1.237.0
```

Comment:
For node 20 the version should be `1.237.1` @kirill-ivlev to confirm

Reply:
The version information should automatically managed by the code generation.    -- @FinVamp1 did you manually change the version numbers to get in this state?  Might be a bug in the version generator -- I'll take a look

Reply:
Finbar and I got the version issues sorted.

Reply:
I'll follow up on validation for dup versions

Reply:
Resolved

Reply:
PR to add Validation: #19623